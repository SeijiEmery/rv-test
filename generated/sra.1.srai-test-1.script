load /x 0 generated/sra.1.srai-test-1.hex
writereg 1 1
writereg 2 15
writereg 3 16
writereg 4 31
writereg 5 30
writereg 6 44
writereg 7 5
setpc 0
run 1004
readreg 1
readreg 2
readreg 3
readreg 4
readreg 5
readreg 6
readreg 7