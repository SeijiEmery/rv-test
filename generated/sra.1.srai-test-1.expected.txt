R1 = 0
R2 = 7
R3 = 8
R4 = 15
R5 = 15
R6 = 22
R7 = 2
